kola/tests/bpf/bpf.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // cmdPrefix is a temporary hack to pull `bcc` tools into Flatcar
-const cmdPrefix = "docker run -d --name %s -v /lib/modules:/lib/modules -v /sys/kernel/debug:/sys/kernel/debug -v /sys/fs/cgroup:/sys/fs/cgroup -v /sys/fs/bpf:/sys/fs/bpf --privileged --net host --pid host quay.io/iovisor/bcc %s"
+const cmdPrefix = "docker run -d --name %s -v /lib/modules:/lib/modules -v /sys/kernel/debug:/sys/kernel/debug -v /sys/fs/cgroup:/sys/fs/cgroup -v /sys/fs/bpf:/sys/fs/bpf --privileged --net host --pid host ghcr.io/iovisor/bcc %s"
```

Comment:
```suggestion
const cmdPrefix = "docker run -d --name %s -v /lib/modules:/lib/modules -v /sys/kernel/debug:/sys/kernel/debug -v /sys/fs/cgroup:/sys/fs/cgroup -v /sys/fs/bpf:/sys/fs/bpf --privileged --net host --pid host ghcr.io/flatcar/bcc %s"
```
This org's registry or?

Reply:
This change can also be made in a subsequent PR after the build and push has been done.

Reply:
I think it's better now

Reply:
> I think it's better now because image building takes some time and we need to merge the PRs for the ref bump before it's used

we also wait for the new mangle image build to finish before we update the container refs in the scripts repo. Nothing will immediately break by merging this here with `ghcr.io/flatcar`

Reply:
updated with the correct `ghcr.io/flatcar` -> this is what I wanted to write initially (somehow it got rewritten by my force push).